util/atom/atom.go

```diff
@@ -1,6 +1,7 @@
 package atom
 
-// An uncoordinated, atomic reference cell in the style of Clojure.
+// Atom are uncoordinated, atomic reference cells.
+// Inspired by <https://clojure.org/reference/atoms>.
```

Comment:
So are you just rewriting Clojure now? haha

Reply:
I mean, Clojure got sooo so much stuff right.